scripts/test/unit-with-coverage

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -eu -o pipefail
+
+for pkg in $(go list ./... | grep -v /vendor/); do
+    ./scripts/test/unit \
+        -cover \
+        -coverprofile=profile.out \
+        -covermode=atomic \
+        ${pkg}
+    
+    if test -f profile.out; then
```

Comment:
What is "this"? The `profile.out` filename? 

https://lk4d4.darth.io/posts/multicover/